types/stripe/index.d.ts

```diff
@@ -411,7 +411,7 @@ declare namespace Stripe {
                  * "individual" or "company". This field is required when attaching the
                  * bank account to a customer object.
                  */
-                account_holder_type?: string;
+                account_holder_type?: "individual" | "company" | null;
```

Comment:
https://stripe.com/docs/api/customer_bank_accounts/object#customer_bank_account_object-account_holder_type

Stripe will set this to `null` in responses if the value is not set.